pom.xml

```diff
@@ -90,7 +90,7 @@ THE SOFTWARE.
     <project.patchManagement.url>https://api.github.com</project.patchManagement.url>
     <patch.tracker.serverId>jenkins-jira</patch.tracker.serverId>
 
-    <matrix-auth.version>2.6.6</matrix-auth.version>
+    <matrix-auth.version>2.6.7</matrix-auth.version>
```

Comment:
This version is used both in the `test/` directory (where bumping the version seems fine) and the list of bundled (detached) plugins in the `war/` directory. I am not sure whether or not it is fine to bump the latter version. I was previously cautioned not to do this in [#5244 (comment)](https://github.com/jenkinsci/jenkins/pull/5244#issuecomment-786142004), but since then we have bumped bundled plugin versions in #5330 and #5347. I am still trying to better understand when it is and is not appropriate to bump bundled plugin versions.

Reply:
@jglick to explain the general approach we want to take here I suppose, given one of his explanations was referenced.

FWIW this specific change should be not particularly problematic -- I hope -- just cleaning up some JS. Could be considered vaguely related to [JENKINS-60865](https://issues.jenkins.io/browse/JENKINS-60865). A larger change is coming though.

Reply:
I guess we should remove this POM property and decouple the version we bundle vs. the version we use for functional tests. (Better yet, remove it from functional tests entirely. Generally we should be using `MockAuthorizationStrategy`.)